flutter.el

```diff
@@ -184,14 +233,30 @@ args."
   (let ((test-file (file-relative-name buffer-file-name (flutter-project-get-root))))
     (flutter--test test-file)))
 
+;;;###autoload
+(defun flutter-test-at-point ()
+  "Execute `flutter test --plain-name <test-name-at-point> <current-file>` inside Emacs."
+  (interactive)
+  (let ((test-file (file-relative-name buffer-file-name (flutter-project-get-root)))
+        (line (line-number-at-pos (point))))
+    (flutter--test
+     "--plain-name"
+     (format "'%s'" (flutter--find-test-case line))
+     test-file)))
+
 ;;;###autoload
 (define-derived-mode flutter-mode comint-mode "Flutter"
   "Major mode for `flutter-run'.
 
 \\{flutter-mode-map}"
   (setq comint-prompt-read-only t))
 
+(defun flutter-test-enable ()
+  "Enable the flutter test mode."
+  (flutter-test-mode t))
+
 (add-hook 'flutter-mode-hook #'flutter--initialize)
+(add-hook 'dart-mode-hook #'flutter-test-enable)
```

Comment:
I think we don't need this part:

1. The `flutter-test-mode` function is itself an adequate toggle
2. I don't know if people will really want to turn it on all the time. Or if they necessarily even have `dart-mode` (though it seems likely they would).

I would rather advertise the existence of `flutter-text-mode` and offer a configuration suggestion in the readme.

Reply:
Nice, It makes sence